src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant1IntendToSwitchToSoleFoNotification.java

```diff
@@ -40,4 +46,18 @@ public void sendToApplicant2(CaseData data, Long id) {
             data.getApplicant2().getLanguagePreference()
         );
     }
+
+    @Override
+    public void sendToApplicant2Solicitor(CaseData data, Long id) {
+        if (YES.equals(data.getFinalOrder().getDoesApplicant1IntendToSwitchToSole())) {
```

Comment:
do you even get to this point if they say no?

Reply:
It doesn't look like it, removed

Reply:
It looke like it's only called from CitizenIntendToSwitchToSoleFO and the decision is made there